### PR TITLE
Add puebla@rladies.org

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Comment by yabellini** [comment]
@gdequeiroz, can you see if  I resolved the conflict correctly? For what I saw, when I add the email were not meetup account in the info, but before you try to merge my PR, another user updates the meetup account and was merge before my changes which causes the conflict. Thank you!!

File: Current-Chapters.csv [hunk 1]
```diff
@@ -108,7 +108,8 @@
 "Luxembourg","","Luxembourg","","","","","","","","","","","Claudia Albanese","Prospective"
 "Mexico","","Mexico City","https://www.meetup.com/rladies-cdmx/","https://twitter.com/RLadiesCDMX","cdmx@rladies.org","https://www.facebook.com/rladiescdmx","","","","","","rladiescdmx.slack.com","Silvia Eunice Gutiérrez de la Torre, Maria Ortiz, Mariana Carmona, Mariana Godina","Active"
 "Mexico","","Morelia","","","","","","","","","","","Thomai Tsiftsi","Prospective"
-"Mexico","","Puebla","","","puebla@rladies.org","","","","","","","","Averi Giudicessi","Prospective"
+yabellini-patch-42
```
Comment (gdequeiroz) [inline Current-Chapters.csv]: @yabellini - Not sure why it is showing `yabellini-patch-42`

Comment (yabellini) [reply Current-Chapters.csv]: Do I have to do something??

Comment (yabellini) [reply Current-Chapters.csv]: I deleted that line. Now don't show any conflict.

**Comment by cvitolo** [comment]
Hi @yabellini, I'm going to close this PR because the changes here were committed directly to master. Thank you!

**Pull request closed without merging**